src/averageAges.js

```diff
@@ -1,5 +1,42 @@
 'use strict';
 
+const getByKeyValue = (key, value) => people => {
+  const peopleCopy = [ ...people ];
+
+  return peopleCopy.filter(person => person[key] === value);
+};
+
+const getMen = getByKeyValue('sex', 'm');
+const getWomen = getByKeyValue('sex', 'f');
+
+const getValuesArr = (key) => people => {
+  const resultArr = [];
+
+  people.filter(person => person[key] !== null
+    ? resultArr.push(person[key]) : null);
+
+  return resultArr;
+};
+
+const getMotherNames = getValuesArr('mother');
+
+const getMothers = (people, group = people) => {
+  const women = getWomen(people);
+  const motherNames = getMotherNames(group);
+  const mothers = women.filter(woman => motherNames.includes(woman['name']));
+
+  return mothers;
+};
+
+const getCentury = died => Math.ceil(died / 100);
+
+const getAges = people => people.map(person => person['died'] - person['born']);
+
+const getAverage = ages => +(
+  ((ages.reduce((acc, cur) => acc + cur, 0)) / ages.length
+  ).toFixed(2)
+) || 0;
```

Comment:
```suggestion
const getAverage = ages => (
  ages.reduce((acc, cur) => acc + cur, 0) / ages.length
);
```

Reply:
I reduced this code-line accordingly your suggestions
I had some problems with tests
and written this after I've seen test log like `expected: 67.68  ... received...`

I accepted your variant - it pass all test successfully

so thank you